apps/site/lib/site_web/views/mode_view.ex

```diff
@@ -25,6 +25,7 @@ defmodule SiteWeb.ModeView do
 
   def get_route_group(route_type, route_groups), do: route_groups[route_type]
 
+  # TODO: Remove this once it is no longer being used in the content on any environment -- MSS 20200730
```

Comment:
Looks like this is making credo fail 😬

Reply:
Thanks! I missed that, but fixed it now (as well as a broken test).